src/App.tsx

```diff
@@ -1,173 +1,117 @@
-                      Jan Frans van Brussel
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+              Home
+            </NavLink>
+            <NavLink
+              className={({ isActive }) => (
+                cn(
+                  'navbar-item',
+                  {
+                    'has-background-grey-lighter': isActive,
+                  },
+                )
+              )}
+              to="/people"
+            >
+              People
+            </NavLink>
           </div>
         </div>
-      </div>
-    </main>
-  </div>
-);
+      </nav>
+
+      {isHomePage && <Navigate to="/" replace />}
+      <Routes>
+        <Route
+          path="/"
+          element={<HomePage />}
+        />
+        <Route
+          path="/people"
+        >
+          <Route
+            index
+            element={(
+              <People
+                people={people}
+                isLoading={isLoading}
+                loadingError={loadingError}
+                setSelectedPerson={setSelectedPerson}
+                selectedName={selectedName}
+                setSelectedName={setSelectedName}
+              />
+            )}
+          />
+          <Route
+            path={`/people/:${selectedPerson}`}
```

Comment:
The path should be '/people/:personSlug' instead of using the selectedPerson variable. The dynamic part of the path should be a descriptive parameter name.